contents/javascript/Javascript-promises.md

```diff
@@ -0,0 +1,104 @@
+# Javascript: Promises
+
+Author: Daniel Berzin Chua
+
+In Javascript, there are occasions where a method call may not resolve instantly and as such, the code may not run in the sequence that you would expect due this asynchronous nature. It can be especially confusing for beginners learning Javascript.
```

Comment:
I wouldn't call it a "method call". perhaps you can say that an operation is executed asynchronously and its result may not be obtained immediately as a function return value would.